docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -278,6 +274,14 @@ Creates an exception that excludes all LFC-signed trusted processes:
 [role="screenshot"]
 image::images/nested-exp.png[]
 
+[float]
+[[rules-using-same-exception]]
+=== Find rules using the same exception
+From the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item and click *Affects X rules* to display the rules affected by the exception.
```

Comment:
This section could use a little more context, maybe an extra sentence easing the reader into the idea of related/affected rules before jumping into actual instructions. Something like "If an exception is used by other rules, this is indicated on the exception" (not really an elegant sentence, but that general idea).

Reply:
How about:

```suggestion
An exception might be used by other rules. To find out, click on the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item, then click *Affects X rules*. 
```

Reply:
This is good -- I like `select` vs. `click on`, especially when the user has a choice. I also think `X` is supposed to be in italics, (_x_), but that may not render with the bold already there -- can't remember. It may be OK. 

You could also flip this if you wanted: 

`"To find out if an exception is used by other rules...." `

Reply:
```suggestion
To find out if an exception is used by other rules, select the *Rule exceptions* or *Endpoint exceptions* tab, navigate to an exception list item, then click *Affects _X_ rules*. 
```